### PR TITLE
added dark theme to querymanager dropdown lists

### LINKED ISSUE #1745: Dark theme not applied to QueryManager
### Is your proposal related to a problem?

The dark theme is not applied to the QueryManager dropdown list:
image

(Write your answer here.)

### Describe the solution you'd like

We should change the white background in the dropdown to a darker shade like here:
image

### Describe alternatives you've considered

(Write your answer here.)

### If the feature is approved, would you be willing to submit a PR?
Yes

### PR DESCRIPTION
**Pull request by karlrez**
Resolves #1745 

No changes to functionality, just added dark theme.

Datasource dropdown:
image
image

Operation dropdown:
image
image

### CODE REVIEW

**Comment by karlrez** [comment]
Pushed again as I noticed the width of the Operation dropdown did not match light mode. This is fixed now:

image


I did add quite a bit to the theme.scss file. Would you prefer if I added a seperate file for the styling? Let me know of any suggestions.

**Comment by Navaneeth-pk** [comment]
@arpitnath can you please review this PR?

**Comment by arpitnath** [comment]
@karlrez 
Here the svgs are inverted and some of the icons are not legible for a user to understand.
Since we can rename the datasources, proper visibility of the icons is necessary.
But yes we need to update the background accordingly to the theme.
Screenshot 2022-01-18 at 2 12 34 AM

Can you please revert the changes for the icons for now and we are good to go 😅

File: frontend/src/_styles/theme.scss [hunk 1]
```diff
@@ -1086,6 +1089,219 @@ button {
...
+  text-align: left;
+  cursor: pointer;
+  border-radius: 0;
+}
+
+.select-search-dark--multiple .select-search-dark__option {
+  height: 48px;
+}
+
+/**
```
Comment (arpitnath) [inline frontend/src/_styles/theme.scss:1206]: Also can you remove the unused commented parts

**Review by arpitnath — CHANGES_REQUESTED** [review]

**Comment by karlrez** [comment]
Hi @arpitnath, thank you for your review and suggestions, I've just pushed my updates.

I agree, icons look better this way:
image

**Review by Navaneeth-pk — APPROVED** [review]

**Pull request merged**